.github/workflows/tests.yml

```diff
@@ -0,0 +1,40 @@
+name: tests
```

Comment:
Maybe make sure that this PR only runs when a change has been made to Python code, so that when we change some data file or a Github Action, the tests dont run again (see example in the other GH Actions)

Reply:
done